homeassistant/components/zwave_js/manifest.json

```diff
@@ -8,6 +8,7 @@
   "integration_type": "hub",
   "iot_class": "local_push",
   "loggers": ["zwave_js_server"],
+  "quality_scale": "platinum",
```

Comment:
I'm not sure we have proper exception handling in all service handlers. Please review all of those.

https://developers.home-assistant.io/docs/integration_quality_scale_index#silver-

Reply:
thx, forgot to submit a PR for that